src/components/assets/EvmToken.vue

```diff
@@ -27,16 +27,42 @@
               </div>
             </div>
           </div>
-          <div class="column--asset-buttons column--buttons--2-columns">
-            <button class="btn btn--sm bg--astar color--astar">
+          <div class="column--asset-buttons column--buttons--multi">
+            <button
+              class="btn btn--sm bg--astar color--astar"
+              @click="
+                handleModalTransfer({
+                  isOpen: true,
+                  currency: token.symbol === tokenSymbol ? tokenSymbol : token,
```

Comment:
`token.symbol === tokenSymbol ? tokenSymbol : token` is very hard to read since without context, it feels like you are checking for token symbol with token symbol, and assigning either the token or the symbol.
Please change the variable name to something that is easier to understand

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/291/commits/8a1ddf4b983c09d126ea500ad05556c8f8b8e4fd) commit